# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/ayatana-indicator-messages/hotfix-doc.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- ayatana-indicator-messages-24.5.1/CMakeLists.txt.vanilla	2026-08-08 21:13:41.788925726 +0300
+++ ayatana-indicator-messages-24.5.1/CMakeLists.txt	2026-08-08 21:13:45.348431906 +0300
@@ -69,7 +69,7 @@
 add_subdirectory(libmessaging-menu)
 add_subdirectory(data)
 add_subdirectory(po)
-add_subdirectory(doc)
+#add_subdirectory(doc)
 
 # Testing and coverage
 

